tools/chapel-py/src/core-types-pernode.cpp

```diff
@@ -75,19 +76,53 @@ static const char* intentToString(IntentType intent) {
   return qualifierToString(Qualifier(int(intent)));
 }
 
-static const resolution::ResolvedExpression*
-scopeResolveResultsForNode(Context* context, const AstNode* node) {
+static const ID scopeResolveVisibilityStmt(Context* context, const AstNode* visibilityStmt, const AstNode* node) {
```

Comment:
Giving it some thought, all of these should be called `scopeResolveViaX`, not `scopeResolveX`. They don't _just_ scope resolve the target, but they lookup `node` in the resolution results.